src/web/client/context/fileData.ts

```diff
@@ -95,7 +104,8 @@ export class FileData implements IFileData {
         encodeAsBase64?: boolean,
         mimeType?: string,
         isContentLoaded?: boolean,
-        logicalEntityName?: string
+        logicalEntityName?: string,
+        logicalFormName?: string
```

Comment:
we should not make this another parameter here - as its not generic parameter used by all entities - instead this should go in an interface with entityMetadata that can be expanded later - keep this one as a nullable param in that interface.